src/collections/events/ccoss2020/index.mdx

```diff
@@ -0,0 +1,8 @@
+---
+title: "CCOSS 2020"
+subtitle: ""
+date: 2020-10-19
```

Comment:
@Jashpatel1 Shouldn't the format of the date be something like `date: 2018-03-21 12:15:05 +0000` this?

Reply:
I dont guess it would be required here, as multiple events on the same day are not present right now. But for future and to have consistency we can use the same format as you mentioned. @vinayaksh42 please change this.

Reply:
I think the time is not required as it won't be reflected on the card also the information of time is not given at the layer5.io site.

Reply:
added @kishore007k